### ISSUE TITLE
Can No Longer Import Data

### ISSUE THREAD

**Issue by Zakkumaru** [ramack/ActivityDiary#281]
This is a separate issue from #280.

Basically, I can no longer import my database back into ActivityDiary.  I've been trying all different methods, all day.  Before, I used to have to toy with the activities, setting new ones or deactivating, etc. to get things to clear and show the database I imported.  Now, I can no longer get it to import, at all.

Things I've tried:

- Uninstalling, reinstalling, importing
- Killing the app, clearing cache, clearing data, importing
- Killing the app, clearing cache, importing
- Using root to delete all traces; trying the above, again
- Using root to manually replace the data in `/data/data/...`

Nothing's working.

**Comment by ramack** [comment]
What does it mean 'isn't working'? Is there any error message? Do the activities or the history not show up or both? Did you try closing and restarting after import?

**Comment by Zakkumaru** [comment]
> - Killing the app, clearing cache, clearing data, importing
> - Killing the app, clearing cache, importing
Yeah, I killed the app, multiple times.  I've restarted the phone, multiple times.

Yeah, neither the history nor activities show up.  It basically looks like I never clicked "Import".  It says it's successful, but I think it just says that no matter what.

**Comment by ramack** [comment]
very strange. Right now I cannot reproduce this, in case you have an export showing the issue (and doesn't have privacy related data) at hand, don't hesitate to share it with me privately.

**Comment by Zakkumaru** [comment]
I think it's possibly related to how the app does cache, or something, since doing a straight file swap still doesn't update the contents of the database, even after killing the app and restarting the phone.
